application/src/test/java/org/thingsboard/server/controller/TwoFactorAuthConfigTest.java

```diff
@@ -472,4 +493,45 @@ public void testDeleteTwoFaAccountConfig() throws Exception {
                 .doesNotContainKey(TwoFaProviderType.SMS);
     }
 
+    @Test
+    public void testDeleteAdminSettingsWithTransactionalOk() throws Exception {
+        AccountTwoFaSettings accountTwoFaSettings = createAccountTwoFaSettings();
+        Assert.isTrue(accountTwoFaSettings.getConfigs().size() > 0);
+
+        twoFaConfigManager.deletePlatformTwoFaSettings(tenantId);
+
+        AccountTwoFaSettings accountTwoFaSettingsAfter = readResponse(doGet("/api/2fa/account/settings").andExpect(status().isOk()), AccountTwoFaSettings.class);
+        Assert.isTrue(accountTwoFaSettingsAfter.getConfigs().size() == 0);
```

Comment:
isEmpty

Reply:
Java  assert:  have not isEmpty

Reply:
> 

Assert.isTrue(accountTwoFaSettingsAfter.getConfigs().**isEmpty()**)